### ISSUE TITLE
[Category] optifine

### ISSUE THREAD

**Issue by ZoruaBit** [sp614x/optifine#5146]
## Description of Issue
RT shaders dont work with the mod create

## Steps to Reproduce
download PTGI and then place down blocks

## OptiFine Version
G5 for 1.15

## Installation Method
dropped the jar in the mods folder

## Fabric/Forge Version
forage 35.1.4

## OptiFabric Version
**If OptiFine is installed as standalone, or you're using Forge, you can remove this section.**

## Other Installed Mods
Create  (This is it)

## Log Files/Crash Reports
No relevent logs. as all the logs dont have anything except me joining and then leaving the world

## F3 Debug Screenshot
![image](https://user-images.githubusercontent.com/39221658/103162719-89b82280-47b9-11eb-84c0-4af03aac8198.png)

## Prior Testing

If you've installed using Forge/Fabric:
This only happens with raytracing shaders (shown with ptgi) neather removing optifine or the mod will cause this as bolth of those elemenate the problem.
## Additional Information
Normal shaders work completly fine. this is using amd.

**Comment by ZoruaBit** [comment]
i casually forgot to remove the optifabric verson. wich was unused

**Comment by Regnander** [comment]
> i casually forgot to remove the optifabric verson. wich was unused

Click the icon with three dots in the upper right corner of your post and then "Edit".

**Comment by sp614x** [comment]
What is the block (plant) that has problems with rendering?

**Comment by ZoruaBit** [comment]
The block is create0.3 1.16 shafts

**Comment by ZoruaBit** [comment]
i have updated the title cause the excat same thing is still happening in the latest 1.16 release.

**Comment by berkkrkc09** [comment]
Yeah, i have weird black dots scattered on all the blocks when ray traced reflections are active. 
1.16.4 - Sildurs Vibrant Shaders Lite - v1.281

**Comment by ZoruaBit** [comment]
> 
> 
> Yeah, i have weird black dots scattered on all the blocks when ray traced reflections are active.
> 1.16.4 - Sildurs Vibrant Shaders Lite - v1.281
is this with astral sorcery? if so that mod and shaders doesnt have great compatibiltiy and has been that way for most of astral sorcerys lifetime. the mod i am complaning about is create wich has pritty good shader compactibility

**Comment by ZoruaBit** [comment]
@sp614x i have an update. this seems to be an AMD only bug. can you please fix it soon ;-;
